### DISCUSSION TITLE
子应用占用内存未清除干净？

### DISCUSSION THREAD

**Discussion by Z-Champion** [umijs/qiankun#1928] [Q&A]
从chrome 性能快照来看，当打开主应用时内存占用20M，打开子应用内存占用达120M左右，关闭子应用后，内存来到60M。请问是不是（猜想：沙箱机制占用内存的在子应用卸载时未清理赶紧）内存未清理干净造成的？

**Comment by eightHundreds** [comment]
你把devtool关掉试试? 用浏览器的进程管理看内存
